docs/bridge.md

```diff
@@ -176,16 +176,16 @@ Replace `localhost:9100` in the redirect URIs with your the `ocis-phoenix` host
 
 #### Run it!
 
-You can now bring up `ocis-connectd` with:
+You can now bring up `ocis-konnectd` with:
 ```console
-$ bin/ocis-konnectd server --iss https://192.168.1.100:9130 --identifier-registration-conf assets/identifier-registration.yaml --signing-kid gen1-2020-02-27
+$ bin/ocis-konnectd server --iss https://192.168.1.100:9130 --identifier-registration-conf assets/identifier-registration.yaml --signing-kid gen1-2020-02-27 --tls=false
```

Comment:
the iss should point to the ocis-proxy if we want to hide everything behind it:
`--iss https://localhost:9200`